apps/DDnet/install

```diff
@@ -1,7 +1,15 @@
 #!/bin/bash
 
 #Dependencies
-install_packages build-essential cmake git libcurl4-openssl-dev libssl-dev libfreetype6-dev libglew-dev libnotify-dev libogg-dev libopus-dev libopusfile-dev libpnglite-dev libsdl2-dev libsqlite3-dev libwavpack-dev python google-mock || exit 1
+install_packages build-essential cmake git libcurl4-openssl-dev libssl-dev libfreetype6-dev libglew-dev libnotify-dev libogg-dev libopus-dev libopusfile-dev libpnglite-dev libsdl2-dev libsqlite3-dev libwavpack-dev python google-mock glslang-tools glslang-dev libx264-dev || exit 1
```

Comment:
I believe you've added glslang-tools glslang-dev dependencies here by mistake.

Reply:
I already corrected it before your comment